lib/CModHandler.h

```diff
@@ -210,18 +240,6 @@ class DLL_LINKAGE CModInfo
 	static std::string getModDir(std::string name);
 	static std::string getModFile(std::string name);
 
-	template <typename Handler> void serialize(Handler &h, const int version)

```

Comment:
No longer needed?

Reply:
No, just mod name and version are serialised